django/src/rdwatch/views/site_evaluation.py

```diff
@@ -37,47 +39,72 @@ def get_responses(self, *args, **kwargs):
 @api_view(["GET"])
 @schema(SiteEvaluationsSchema())
 def site_evaluations(request: HttpRequest):
-    queryset = (
-        SiteEvaluationsFilter(
-            request.GET, queryset=SiteEvaluation.objects.order_by("-timestamp")
-        )
-        .qs.annotate(
-            timemin=Min("observations__timestamp"),
-            timemax=Max("observations__timestamp"),
-        )
-        .aggregate(
-            count=Count("pk"),
-            timerange=JSONObject(
-                min=ExtractEpoch(Min("timemin")),
-                max=ExtractEpoch(Max("timemax")),
-            ),
-            bbox=BoundingBox(Collect("geom")),
-            results=JSONBAgg(
-                JSONObject(
-                    id="pk",
-                    site=JSONObject(
-                        region=JSONObject(
-                            country="region__country",
-                            classification="region__classification__slug",
-                            number="region__number",
-                        ),
-                        number="number",
-                    ),
-                    configuration="configuration__parameters",
-                    performer=JSONObject(
-                        team_name="configuration__performer__description",
-                        short_code="configuration__performer__slug",
-                    ),
-                    score="score",
-                    bbox=BoundingBox("geom"),
-                    timestamp=ExtractEpoch("timestamp"),
-                    timerange=JSONObject(
-                        min=ExtractEpoch("timemin"),
-                        max=ExtractEpoch("timemax"),
+    queryset = SiteEvaluationsFilter(
+        request.GET,
+        queryset=SiteEvaluation.objects.order_by("-timestamp"),
+    ).qs
+
+    # Pagination
+    assert api_settings.PAGE_SIZE, "PAGE_SIZE must be set."
+    paginator = Paginator(queryset, api_settings.PAGE_SIZE)
```

Comment:
We could also accept a `page_size` query param here, and just treat `api_setting.PAGE_SIZE` as a maximum page size/default if the parameter is invalid. @mcovalt should I add that?

Reply:
Good idea. Maybe in the future, but no need to include it here.